src/qcodes/instrument_drivers/cryomagnetics/Cryomagnetics4G_visa.py

```diff
@@ -0,0 +1,287 @@
+import re
```

Comment:
I don't  think there is a good reason for including visa in the name of this file. That only happed for the AMI driver because we had two versions

Reply:
Good point, I'll rename this module to not include _visa.